vsm

```diff
@@ -15,6 +15,8 @@ import argparse
 import yaml
 import ast
 
+from ipc.loader import LoaderError
```

Comment:
Makes no sense to import one thing here and the other at the bottom. Move everything here, perhaps?